lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -771,29 +792,20 @@ def create_key(connection, module):
         module.fail_json_aws(e, msg="Failed to create initial key")
     key = get_key_details(connection, module, result['KeyId'])
 
-    alias = module.params['alias']
-    if not alias.startswith('alias/'):
-        alias = 'alias/' + alias
-    try:
-        connection.create_alias(AliasName=alias, TargetKeyId=key['key_id'])
-    except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
-        module.fail_json_aws(e, msg="Failed to create alias")
+    update_alias(connection, module, key, module.params['alias'])
 
-    ensure_enabled_disabled(connection, module, key)
-    for grant in module.params.get('grants'):
-        grant_params = convert_grant_params(grant, key)
-        try:
-            connection.create_grant(**grant_params)
-        except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
-            module.fail_json_aws(e, msg="Failed to add grant to key")
+    ensure_enabled_disabled(connection, module, key, True)
```

Comment:
Should True be module.params['enabled']? It looks like before, enabling a created key was optional.